### PR TITLE
failed-test-with-jest-fixed

### PR DESCRIPTION
**Pull request by MajorReact**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I ran the sort command to make sure all terms are sorted
- [ ] I have made corresponding changes to the ReadMe
- [ ] My changes generate no new warnings
- [ ] Existing unit test passes locally with my changes

### CODE REVIEW

File: data/terms.json [hunk 1]
```diff
@@ -729,8 +728,8 @@
     "meaning": "The 'read-write web,' which began in the 1990s, is distinguished by user-generated content and enhanced user interfaces. Web2 emphasizes user experience and interoperability across different applications and websites, resulting in an extensive network of linked websites."
   },
   {
-    "term": "web2.0",
-    "meaning": "The 'read-write web,' which began in the 1990s, is distinguished by user-generated content and enhanced user interfaces. Web2 emphasizes user experience and interoperability across different applications and websites, resulting in an extensive network of linked websites."
+    "term": "web2",
+    "meaning": "I love the JavaScript ecosystem as it is used both in the web 2.0 as well as web 3.0. If you are a javascript developer, stick to the language and get better at it. It's flexibilty and extensibility across the board cannot be overemphasized."
```
Comment (Qudusayo) [inline data/terms.json:732]: This definition seemed to be more of a joke

Comment (Olanetsoft) [reply data/terms.json:732]: > This definition seemed to be more of a joke

Exactly, not sure what he is trying to say @MajorReact

Comment (MajorReact) [reply data/terms.json:732]: > > This definition seemed to be more of a joke
> 
> Exactly, not sure what he is trying to say @MajorReact

Sorry about the weird headline. I failed some tests while trying to make some contributions and earn some badges at the #hacktoberfest. The prebuilt tests already set, help me understand testing more. It's my first-time using jest and I gained some experience while making contributions. My name is Nnamdi from Nigeria by the way. I follow you on twitter and I saw you shared the repo for others to contribute.

Comment (MajorReact) [reply data/terms.json:732]: > > This definition seemed to be more of a joke
> 
> Exactly, not sure what he is trying to say @MajorReact

Will it be accepted?

**Pull request closed without merging**